src/k8s-engine/wsl.ts

```diff
@@ -688,10 +717,11 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
       options = optionsOrArg;
     }
     try {
-      const stream = await Logging['wsl.exec'].fdStream;
+      const stream = options.logStream ?? await Logging['wsl.exec'].fdStream;
 
       // We need two separate calls so TypeScript can resolve the return values.
       if (options.capture) {
+        console.debug(`Capturing output: wsl.exe ${ args.join(' ') }`);
```

Comment:
Ahh, nice, we have `console.debug` now and don't have to worry about managing them as much.